runtime/moonriver/src/lib.rs

```diff
@@ -924,11 +924,18 @@ impl pallet_proxy::Config for Runtime {
 
 impl pallet_migrations::Config for Runtime {
 	type Event = Event;
-	type MigrationsList = runtime_common::migrations::CommonMigrations<
-		Runtime,
-		CouncilCollective,
-		TechCommitteeCollective,
-	>;
+	type MigrationsList = (
+		runtime_common::migrations::CommonMigrations<
+			Runtime,
+			CouncilCollective,
+			TechCommitteeCollective,
+		>,
+		runtime_common::migrations::XcmMigrations<
```

Comment:
Just to be pedantic, this means we can order `CommonMigrations` and `XcmMigrations` independently, and that all `XcmMigrations` come after all `CommonMigrations`.

This is possible because of the `impl GetMigrations for Tuple`.

I believe this will cause a lot of allocation and copies every time `get_migrations()` is called, but that shouldn't happen outside of runtime upgrades.